### PR TITLE
(DOCSP-16624) Backfills docs content for 5 clusters search indexes commands

### PR DESCRIPTION
**Pull request by sarahsimpers**
## Proposed changes

Backfills docs content for 5 atlas clusters search indexes commands. Also updates a few flag text entries for clarification.

_Jira ticket:_
https://jira.mongodb.org/browse/DOCSP-16624
https://jira.mongodb.org/browse/DOCSP-16625
https://jira.mongodb.org/browse/DOCSP-16626
https://jira.mongodb.org/browse/DOCSP-16627
https://jira.mongodb.org/browse/DOCSP-16628

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works **N/A**
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added) **N/A**
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by gssbzn — APPROVED** [review]
LGTM

File: docs/atlascli/command/atlas-accessLists-list.txt [hunk 1]
```diff
@@ -41,15 +41,15 @@ Options
    * - --limit
      - int
      - false
-     - Number of items per page. This value defaults to 100.
+     - Number of items per page of results. This value defaults to 100.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-accessLists-list.txt:44]: Suggestion for clarity:
```suggestion
     - Number of items per results page. This value defaults to 100.
```

File: docs/atlascli/command/atlas-clusters-search-indexes-delete.txt [hunk 2]
```diff
@@ -12,7 +12,7 @@ atlas clusters search indexes delete
    :depth: 1
    :class: singlecol
 
-Delete a search index from a cluster.
+Delete the search index you specify from a cluster.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-clusters-search-indexes-delete.txt:15]:
```suggestion
Delete the specified search index from the specified cluster.
```

File: docs/atlascli/command/atlas-clusters-search-indexes-list.txt [hunk 3]
```diff
@@ -12,7 +12,7 @@ atlas clusters search indexes list
    :depth: 1
    :class: singlecol
 
-List Atlas search indexes for a cluster.
+List all Atlas search indexes for a cluster.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-clusters-search-indexes-list.txt:15]:
```suggestion
List all Atlas Search indexes for a cluster.
```

File: docs/atlascli/command/atlas-clusters-search-indexes-list.txt [hunk 4]
```diff
@@ -53,15 +53,15 @@ Options
    * - --limit
      - int
      - false
-     - Number of items per page. This value defaults to 100.
+     - Number of items per page of results. This value defaults to 100.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-clusters-search-indexes-list.txt:56]:
```suggestion
     - Number of items per results page. This value defaults to 100.
```

File: docs/atlascli/command/atlas-clusters-search-indexes-update.txt [hunk 5]
```diff
@@ -57,7 +57,7 @@ Options
    * - -f, --file
      - string
      - false
-     - File name to use, file with a json index configuration.
+     - Name of the JSON index configuration file to use. To learn about the format for index configuration files, see https://www.mongodb.com/docs/atlas/atlas-search/index-definitions/.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-clusters-search-indexes-update.txt:60]: We don't have index configuration file format defined anywhere. Suggestion:
```suggestion
     - Name of the JSON index configuration file to use. To learn about the Atlas Search index syntax and options that you can define in your configuration file, see https://www.mongodb.com/docs/atlas/atlas-search/index-definitions/.
```

File: docs/atlascli/command/atlas-clusters-search-indexes.txt [hunk 6]
```diff
@@ -50,10 +50,10 @@ Related Commands
 ----------------
 
 * :ref:`atlas-clusters-search-indexes-create` - Create a search index for a cluster.
-* :ref:`atlas-clusters-search-indexes-delete` - Delete a search index from a cluster.
-* :ref:`atlas-clusters-search-indexes-describe` - Describe a search index for a cluster.
-* :ref:`atlas-clusters-search-indexes-list` - List Atlas search indexes for a cluster.
-* :ref:`atlas-clusters-search-indexes-update` - Update a search index for a cluster.
+* :ref:`atlas-clusters-search-indexes-delete` - Delete the search index you specify from a cluster.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-clusters-search-indexes.txt:53]:
```suggestion
* :ref:`atlas-clusters-search-indexes-delete` - Delete the specified search index from a cluster.
```

File: docs/atlascli/command/atlas-clusters-search-indexes.txt [hunk 7]
```diff
@@ -50,10 +50,10 @@ Related Commands
...
+* :ref:`atlas-clusters-search-indexes-describe` - Return the details for the search index for a cluster.
+* :ref:`atlas-clusters-search-indexes-list` - List all Atlas search indexes for a cluster.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-clusters-search-indexes.txt:55]:
```suggestion
* :ref:`atlas-clusters-search-indexes-list` - List all Atlas Search indexes for a cluster.
```

File: docs/atlascli/command/atlas-customDbRoles-list.txt [hunk 8]
```diff
@@ -41,15 +41,15 @@ Options
    * - --limit
      - int
      - false
-     - Number of items per page.
+     - Number of items per page of results.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-customDbRoles-list.txt:44]:
```suggestion
     - Number of items per results page.
```

File: docs/mongocli/command/mongocli-atlas-clusters-search-indexes-create.txt [hunk 9]
```diff
@@ -57,7 +57,7 @@ Options
    * - -f, --file
      - string
      - false
-     - File name to use, file with a json index configuration.
+     - Name of the JSON index configuration file to use. To learn about the format for index configuration files, see https://www.mongodb.com/docs/atlas/atlas-search/index-definitions/.
```
Comment (kanchana-mongodb) [inline docs/mongocli/command/mongocli-atlas-clusters-search-indexes-create.txt:60]: Same comment

File: docs/mongocli/command/mongocli-atlas-clusters-search-indexes-delete.txt [hunk 10]
```diff
@@ -12,7 +12,7 @@ mongocli atlas clusters search indexes delete
    :depth: 1
    :class: singlecol
 
-Delete a search index from a cluster.
+Delete the search index you specify from a cluster.
```
Comment (kanchana-mongodb) [inline docs/mongocli/command/mongocli-atlas-clusters-search-indexes-delete.txt:15]:
```suggestion
Delete the specified search index from a cluster.
```

File: docs/mongocli/command/mongocli-atlas-clusters-search-indexes-list.txt [hunk 11]
```diff
@@ -12,7 +12,7 @@ mongocli atlas clusters search indexes list
    :depth: 1
    :class: singlecol
 
-List Atlas search indexes for a cluster.
+List all Atlas search indexes for a cluster.
```
Comment (kanchana-mongodb) [inline docs/mongocli/command/mongocli-atlas-clusters-search-indexes-list.txt:15]:
```suggestion
List all Atlas Search indexes for a cluster.
```

File: docs/mongocli/command/mongocli-atlas-clusters-search-indexes-list.txt [hunk 12]
```diff
@@ -53,15 +53,15 @@ Options
    * - --limit
      - int
      - false
-     - Number of items per page. This value defaults to 100.
+     - Number of items per page of results. This value defaults to 100.
```
Comment (kanchana-mongodb) [inline docs/mongocli/command/mongocli-atlas-clusters-search-indexes-list.txt:56]:
```suggestion
     - Number of items per results page. This value defaults to 100.
```

File: internal/cli/atlas/search/delete.go [hunk 13]
```diff
@@ -54,11 +55,13 @@ func DeleteBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "delete <indexId>",
 		Aliases: []string{"rm"},
-		Short:   "Delete a search index from a cluster.",
+		Short:   "Delete the search index you specify from a cluster.",
```
Comment (kanchana-mongodb) [inline internal/cli/atlas/search/delete.go:58]:
```suggestion
		Short:   "Delete the specified search index from the specified cluster.",
```

File: internal/cli/atlas/search/list.go [hunk 14]
```diff
@@ -62,9 +63,11 @@ func ListBuilder() *cobra.Command {
 	opts := &ListOpts{}
 	cmd := &cobra.Command{
 		Use:     "list",
-		Short:   "List Atlas search indexes for a cluster.",
+		Short:   "List all Atlas search indexes for a cluster.",
```
Comment (kanchana-mongodb) [inline internal/cli/atlas/search/list.go:66]:
```suggestion
		Short:   "List all Atlas Search indexes for a cluster.",
```

File: internal/cli/atlas/search/delete.go [hunk 15]
```diff
@@ -54,11 +55,13 @@ func DeleteBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "delete <indexId>",
 		Aliases: []string{"rm"},
-		Short:   "Delete a search index from a cluster.",
+		Short:   "Delete the specified search index from the cluster you specify.",
```
Comment (kanchana-mongodb) [inline internal/cli/atlas/search/delete.go:58]:
```suggestion
		Short:   "Delete the specified search index from the specified cluster.",
```

**Review by kanchana-mongodb — APPROVED** [review]
1 nit, LGTM.

File: internal/cli/atlas/search/create.go [hunk 16]
```diff
@@ -93,6 +94,8 @@ func CreateBuilder() *cobra.Command {
 		Annotations: map[string]string{
 			"indexNameDesc": "Name of the index.",
 		},
+		Example: fmt.Sprintf(`  # Create a search index named myIndex for the cluster named myCluster using a JSON index configuration file named search-config.json:
+  %s clusters search indexes create myIndex --clusterName myCluster --file search-config.json --output json`, cli.ExampleAtlasEntryPoint()),
```
Comment (sarahsimpers) [inline internal/cli/atlas/search/create.go:98]: @gssbzn Can we specify an index name when using a json file? If not I will remove "myIndex"

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**